hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1060,19 +1060,19 @@ public void testNotExposeTableProperties() {
     parameters.put(DEFAULT_PARTITION_SPEC, "partitionSpec");
     parameters.put(DEFAULT_SORT_ORDER, "sortOrder");
 
-    ops.setSnapshotStats(metadata, parameters);
+    HMSTablePropertyHelper.setSnapshotStats(metadata, parameters, maxHiveTablePropertySize);
     assertThat(parameters)
         .doesNotContainKey(CURRENT_SNAPSHOT_SUMMARY)
         .doesNotContainKey(CURRENT_SNAPSHOT_ID)
         .doesNotContainKey(CURRENT_SNAPSHOT_TIMESTAMP);
 
-    ops.setSchema(metadata.schema(), parameters);
+    HMSTablePropertyHelper.setSchema(metadata.schema(), parameters, maxHiveTablePropertySize);
     assertThat(parameters).doesNotContainKey(CURRENT_SCHEMA);
 
-    ops.setPartitionSpec(metadata, parameters);
+    HMSTablePropertyHelper.setSortOrder(metadata, parameters, maxHiveTablePropertySize);
```

Comment:
This should be `HMSTablePropertyHelper.setPartitionSpec`. This is why the test fails.
Please make sure the CI passes!

Reply:
Good catch, thanks!
Fixed!